client/verta/requirements.txt

```diff
@@ -1,3 +1,5 @@
+terminaltables
+
```

Comment:
Unfortunately, the output of `terminaltables` looks a bit outdated to me. I would also prefer to not add a dependency to the Client for a relatively small piece of functionality.

See VR-5448 for an example of what we'd like the output of `list`s to look like!
```suggestion
```